### PR TITLE
Update Member Docs

### PR DESCRIPTION
**Pull request by waitwhatActo**

### CODE REVIEW

**Review by heftymouse — APPROVED** [review]

File: member/index.md [hunk 1]
```diff
@@ -70,7 +71,7 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
...
 | #minecraft-status          | Important notifications regarding the Minecraft server will be posted here.                                                                                                                                                                                               |
-| #minecraft-server            | Use this channel to discuss our official Minecraft SMP server! Details to join are available in the channel description. Note that it is role-gated to members with the Regular role or higher, Microsoft employees or Nitro boosters.                                    |
+| #minceraft-server          | Use this channel to discuss our official Minecraft SMP server! Details to join are available in the channel description. Note that it is role-gated to members with the Regular role or higher, Microsoft employees or Nitro boosters.                                    |
```
Comment (Erisa) [inline member/index.md:74]: Torch said no
```suggestion
| #minecraft-server          | Use this channel to discuss our official Minecraft SMP server! Details to join are available in the channel description. Note that it is role-gated to members with the Regular role or higher, Microsoft employees or Nitro boosters.                                    |
```

File: member/index.md [hunk 2]
```diff
@@ -57,7 +58,7 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
...
 | #music                     | Talk about your favourite music and artists here. You can feel free to share Spotify/YouTube links etc. Occasionally, we may host music listening parties that you can join in with and suggest songs for.                                                                |
 | #tech-support              | Have you got a problem that you’re unable to fix, or just a query that you can’t find the answer for? Ask here! Any support offered here is not official Microsoft support, is offered upon other members' discretion, and may not be accurate or work.                   |
-| #tech-support-forum🆕      | An experimental replacement for #tech-support using Discord's Forum Channels. Allows for more organised discussion in help threads.                                                                                                                                       |
+| #tech-support-forum.       | An experimental replacement for #tech-support using Discord's Forum Channels. Allows for more organised discussion in help threads.                                                                                                                                       |
```
Comment (Erisa) [inline member/index.md:61]:
```suggestion
| #tech-support-forum       | An experimental replacement for #tech-support using Discord's Forum Channels. Allows for more organised discussion in help threads.                                                                                                                                       |
```

File: member/index.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Member Information
+## Member Information
```
Comment (Erisa) [inline member/index.md:1]: Why?

**Review by Erisa — CHANGES_REQUESTED** [review]

File: member/index.md (hunk 3 above)
Comment (waitwhatActo) [reply member/index.md:1]: might be a typo

Comment (Erisa) [reply member/index.md:1]:
```suggestion
# Member Information
```

**Review by Erisa — APPROVED** [review]

**Pull request merged**